File: PRAK205-2510817310005-Tristan_Nathan_Naurell_Prasetyo.c
#include <stdio.h>
#include <math.h>

int main(){
    float a, b, alas, tinggi, keliling, luas;
    scanf("%f", &a);
    scanf("%f", &b);

    alas = sqrt(b*b - a*a);
    tinggi = a;
    keliling = a + b + alas;
    luas = 0.5 * alas * tinggi;

    printf("alas = %.0f cm\n", alas);
    printf("tinggi = %.0f cm\n", tinggi);
    printf("keliling = %.0f cm\n", keliling);
    printf("luas = %.0f cm^2\n", luas);

    scanf("%f", &a);
    scanf("%f", &b);

    alas = sqrt(b*b - a*a);
    tinggi = a;
    keliling = a + b + alas;
    luas = 0.5 * alas * tinggi;

    printf("alas = %.0f cm\n", alas);
    printf("tinggi = %.0f cm\n", tinggi);
    printf("keliling = %.0f cm\n", keliling);
    printf("luas = %.0f cm^2\n", luas);
    return 0;
}

File: PRAK205-2510817310005-Tristan_Nathan_Naurell_Prasetyo.py
import math

a, b = map(float, input().split())

alas = math.sqrt(b**2 - a**2)
tinggi = a
keliling = a + b + alas
luas = 0.5 * alas * tinggi

print("alas = %.0f cm" % alas)
print("tinggi = %.0f cm" % tinggi)
print("keliling = %.0f cm" % keliling)
print("luas = %.0f cm^2" % luas)

a = float(input())
b = float(input())

alas = math.sqrt(b**2 - a**2)
tinggi = a
keliling = a + b + alas
luas = 0.5 * alas * tinggi

print("alas = %.0f cm" % alas)
print("tinggi = %.0f cm" % tinggi)
print("keliling = %.0f cm" % keliling)
print("luas = %.0f cm^2" % luas)
